src/registry/timefield/index.tsx

```diff
@@ -0,0 +1,37 @@
+import type {TimeComponentSchema} from '@open-formulieren/types';
+
+import {TimeField} from '@/components/forms';
+import type {RegistryEntry} from '@/registry/types';
+
+import ValueDisplay from './ValueDisplay';
+import isEmpty from './empty';
+import getInitialValues from './initialValues';
+import getValidationSchema from './validationSchema';
+
+export interface FormioTimeFieldProps {
+  componentDefinition: TimeComponentSchema;
+}
+
+export const FormioTimeField: React.FC<FormioTimeFieldProps> = ({
+  componentDefinition: {key, label, description, tooltip, validate},
+}) => {
+  return (
+    <TimeField
+      name={key}
+      label={label}
+      tooltip={tooltip}
+      description={description}
+      isRequired={validate?.required}
+    />
+  );
+};
```

Comment:
I think we can also do something with passing down the min/max times if they're in the validation configuration